packages/pigeon/e2e_tests/test_objc/ios/Runner/dartle.m

```diff
@@ -0,0 +1,156 @@
+// Autogenerated from Dartle.
```

Comment:
I'm somewhat surprised this is checked in. I would have imagined an e2e test here meant taking an IDL, generating the implementations and then running the implementations :)

Reply:
Good call, removed.  It is getting regenerated if executed through the run_tests.sh.

Reply:
If the issue was the dart linters, could we remove these objective-c files from source control?

Reply:
I think it's confusing to have half your generated code checked in.  It should be one or the other.